### DISCUSSION TITLE
How to render object (not directly variable) from a JSON file

### DISCUSSION THREAD

**Discussion by dl345** [harttle/liquidjs#436] [Q&A]
Hello, I have a question regarding the usage of a engine.render(). 
I have a code like this, which prints an output by using an origin JSON file and a LiquidJS template. 

```
const engine = new Liquid({
    keepOutputType: true
  });

  var testObj = {
    "name" : "washington",
    "profile" : {
      "age" : 19, 
      "nation" : "united states"
    }
  };

  const tpl = engine.parse('Welcome to {{profile}} !');
  engine.render(tpl, testObj).then(console.log);
```

I want the result of this rendering to look like below: 

```
Welcome to {
  "age" : 19,
  "nation" : "united states"
} !
```
That is, I want to print the 'object' inside an origin JSON file, not a variable. 
However, what I see right now with the above code is like this: 

```
Welcome to [object Object]!
```

Is there any way I can match an object inside the origin JSON file by using LiquidJS template? 

Thank You for your support!

**Comment by harttle** [comment]
No, you cannot output the part of original Json file. It's already a
variable before passed into liquidjs.

However, there's a Json filter you can use to format the variable.
https://liquidjs.com/filters/json.html

```
Welcome to {{profile | json}} !
```

Please note the original formatting will be lost.
